taxcalc/functions.py

```diff
@@ -1459,77 +1488,47 @@ def AddCTC(_nctcr, _precrd, c07220, e00200, e82882, e30100, _sey, _setax,
     # Part I of 2005 form 8812
     if _nctcr > 0:
         c82925 = _precrd
-
         c82930 = c07220
-
         c82935 = c82925 - c82930
-
         # CTC not applied to tax
-
-        c82880 = max(0., e00200 + e82882 + e30100 + max(0., _sey) -
```

Comment:
Changes here very likely to be reversed by Dan. Not sure what c82880 is referring to in the form.